src/memote_webservice/resources/submit.py

```diff
@@ -91,12 +92,12 @@ def _load_model(self, file_storage):
         except (CobraSBMLError, ValueError) as err:
             msg = f"Failed to parse model: {str(err)}"
             LOGGER.exception(msg)
```

Comment:
`exception` goes to the error level so we will get a sentry notification every time. Since we now tell users to use the validator or contact us on gitter/GitHub, I'd say this is no longer necessary.

Reply:
So that means we are confident that these exceptions are thrown due to malformed input and not internal bugs?

If we are, it should probably not have had this level all along. But if not, then I'd prefer to keep the internal notification level, as we should not solely rely on users reaching out to discover bugs.

Reply:
Alright, let's keep it for now. If we are reasonably confident in future then yes, I'd say it's a client error. But for now we're not sure so you're right.